pipelines/templates/tasks/upmpackages.yml

```
@@ -21,7 +28,7 @@ steps:
         -ProjectRoot ${{ parameters.projectRoot }}
         -OutputDirectory ${{ parameters.outputDirectory }}
         -Version ${{ parameters.version }}
-        -BuildNumber ${{ parameters.buildNumber }}
+        -BuildNumber $(parameters.previewVersion)
```

I don't think variables need the `parameters.` part

```suggestion
        -BuildNumber $(previewVersion)
```